SingularityService/src/main/java/com/hubspot/singularity/data/SingularityValidator.java

```diff
@@ -1239,6 +1239,22 @@ public void checkScale(SingularityRequest request, Optional<Integer> previousSca
         );
       }
     }
+
+    if (previousScale.isPresent() && !request.getLargeScaleDownAcknowledged()) {
+      boolean scaleDownExceedsAbsoluteMax =
+        previousScale.get() - request.getInstancesSafe() > 10;
```

Comment:
Should we make the 10 configurable? Think that's my only other comment on this one. Easier to change yaml and redeploy than to rebuild